sormas-ui/src/main/java/de/symeda/sormas/ui/importer/DocumentUploadFinishedHandler.java

```diff
@@ -108,8 +110,10 @@ private void saveDocument(
 		String relatedEntityUuid,
 		byte[] bytes)
 		throws Exception {
+		if (isFileSizeLimitExceeded(length))
```

Comment:
would be better to make `FacadeProvider.getDocumentFacade().saveDocument` throw an exception if file is too big so there would be no need to check the file size in several places

Reply:
It cannot be done as the one remark from this feature is asking to upload all files and for failed one to raise a specific dialog where all UUIDs are present. As a generic implementation, same logic was left for a single file.